### PR TITLE
mailtest fix and splitting mailhandler (branch name is a bit misleading)

### PR DESCRIPTION
**Pull request by TheRittler**
web Mailtest now working again and splitted MailHandler into mailSender and mailFetcher in moneytransfer package

### CODE REVIEW

**Comment by LeanderK** [comment]
i would like to see the example properties file for the mail package

**Comment by LeanderK** [comment]
still waiting for an update

**Comment by TheRittler** [comment]
what is your opinion on how i should document that? We should write a documentation.

**Comment by LeanderK** [comment]
what do you want to document?

**Comment by TheRittler** [comment]
The creation and keys/values of the mail config

**Comment by LeanderK** [comment]
make a github wiki-entry (we don't have a uniform format yet) detailing the package and the usage. Also the key/values should be documented.

File: src/main/resources/config.sample.yml [hunk 1]
```diff
@@ -50,8 +53,10 @@ moneytransfer:
     #the password used, to verify a valid amazon giftcode email (enter password as message at amazon homepage, while buying giftcodes
     parsingPassword: test
 
-    #the mail address to send notification mails to
+    #the mail address to send notification mails about moneytransfer issues to
```
Comment (marcelhollerbach) [inline src/main/resources/config.sample.yml]: what ? isnt this the mail adress for notifications anymore?

File: src/main/resources/config.sample.yml [hunk 2]
```diff
@@ -43,6 +43,9 @@ mail :
     # disabled = true will ignore the mail configuration and print to the command-line
     disabled : false
 
+    #specifies the mail address, that gets displayed as sender in moneytransfer mails
+    sendMail: pseipd@gmail.com
```
Comment (marcelhollerbach) [inline src/main/resources/config.sample.yml]: maybe use moneytransferMailAdress, since sendmail is often the acronym for the MDA.

Comment (kelunik) [inline src/main/resources/config.sample.yml]: Better `from` or `sender` or `address`.

File: src/main/resources/config.sample.yml (hunk 1 above)
Comment (TheRittler) [inline src/main/resources/config.sample.yml]: not anymore, it has never been. I dont know which mail address the notification package uses. @phimonk

Comment (LeanderK) [inline src/main/resources/config.sample.yml]: well, we should fix the mess anyway

**Comment by LeanderK** [comment]
You can also update the CommandLineHandler, it has no need to extend MailHandler anymore

**Comment by TheRittler** [comment]
it does run first and then wait, before it started after 7 days, You mean the commandline handler should just implement both interfaces and doesnt extend from MailHandler?

**Comment by TheRittler** [comment]
github wiki entry added, @LeanderK you can look over it, if its that what you wanted.

**Comment by LeanderK** [comment]
please rebase onto master, then it's good to go!

File: src/main/resources/config.sample.yml (hunk 1 above)
Comment (LeanderK) [inline src/main/resources/config.sample.yml]: this must be resolved before merge!

**Pull request merged**